R/utils.R

```diff
@@ -1,11 +1,20 @@
 #'@importFrom httr stop_for_status GET user_agent content status_code
 pv_query <- function(params, reformat = TRUE, ...){
+  if(length(params) > 1){
+    data <- lapply(as.list(params), pv_query, reformat = reformat)
+    if(reformat == TRUE){
+      return(do.call(rbind, data))
+    } else {
+      return(data)
```

Comment:
I took your code as the model for this line (I don't tend to use too much recursion in R) This is just a further abstraction of your code in the article query:

```
  if(length(article) > 1){
    return(lapply(article, article_pageviews, project = project, platform = platform,
                  user_type = user_type, start = start, end = end, ...))
  }
```